src/Catalyst.Core.Lib/Util/CidHelper.cs

```diff
@@ -33,7 +33,7 @@ public static class CidHelper
 
         public static Cid CreateCid(MultiHash multiHash)
         {
-            return new Cid {Version = 1, Hash = multiHash, ContentType = "dag-pb", Encoding = Encoding};
```

Comment:
Is their reasoning behind this change?

Reply:
![image](https://user-images.githubusercontent.com/11431881/68852178-48f16d00-06cf-11ea-87f9-f049eee4404f.png)

I saw dfs/ipfs was returning Content-Type raw not dag-pb.